templates/shared/_blogsection.html

```diff
@@ -9,14 +9,14 @@ <h3><a href="/blog{% if tag_name %}/tag/{{ tag_name }}{% endif %}">Read the late
 
 <section class="u-hide {% if section_classes %}{{ section_classes }}{% else %}p-strip{% endif %}" data-js="latest-news">
```

Comment:
I'm not sure that there's ever a case wherein we want this to be a `p-strip`. We have this on u.com to accommodate the homepage design and legacy blog sections, but it doesn't look like either of those cases are relevant to this codebase